.github/workflows/main.yml

```diff
@@ -22,11 +22,11 @@ jobs:
           GH_TOKEN=${{ secrets.GITHUB_TOKEN }} npm run publish-linux-app
 
   macos:
-    runs-on: macos-10.15
+    runs-on: macos-latest
```

Comment:
This leads to issues such as #98.

Reply:
303aabd0d0668ea8f6ba1a8afee121a78f603d76